src/test/java/com/home/eshop/dao/BikesDaoTest.java

```diff
@@ -0,0 +1,65 @@
+package com.home.eshop.dao;
+
+import com.home.eshop.model.Bike;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BikesDaoTest {
+    private BikesDao bikesDaoTest;
+
+    @BeforeEach
+    void prepare() {
+        bikesDaoTest = new BikesDao();
+        bikesDaoTest.loadData();
+
+    }
+
+    @Test
+    void loadData() {
+        int size1 = bikesDaoTest.loadData();
+        int size2 = bikesDaoTest.bikes.size();
+        assertEquals(size2, size1);
```

Comment:
You should check if the method result equals to you test data, which you prepare in your test

Reply:
use **AssertJ** to test collections

Reply:
add fail tests case

Reply:
Implemented+-